x/examples/outline-connectivity-app/README.md

```diff
@@ -86,21 +86,28 @@ export class AppMain extends LitElement {
 
 ### Prerequisites
 
-- [Node.js](https://nodejs.org/)
-- [Yarn](https://yarnpkg.com/)
-- [Go](https://golang.org/)
-- [Wails](https://wails.app/)
-- [Capacitor](https://capacitorjs.com/)
-- [CocoaPods](https://cocoapods.org/)
-- [Xcode](https://developer.apple.com/xcode/)
-- [Android SDK](https://developer.android.com/studio)
-- [Android NDK](https://developer.android.com/ndk)
-
-### Setup
+- Javascript
+  - [Node.js](https://nodejs.org/)
+  - [Yarn](https://yarnpkg.com/)
+- Golang
+  - [Go](https://golang.org/)
+    - `PATH=$PATH:$(go env GOPATH)/bin`
+  - [Wails](https://wails.app/)
+  - [Gomobile](https://pkg.go.dev/golang.org/x/mobile/cmd/gomobile)
+    - `gomobile init`
```

Comment:
gomobile init is not needed.
I recommend building gomobile and gobind in a single command, then you need to set up the PATH to include gobind when calling gomobile.

We are duplicating instructions though. See https://github.com/Jigsaw-Code/outline-sdk/blob/main/x/mobileproxy/README.md

I've iterated on this for a while, and I found this to be the easiest to do and understand.